Public/Azure/Request-AzureImageGeneration.ps1

```diff
@@ -243,13 +251,15 @@ function Request-AzureImageGeneration {
                 }
 
                 # Download image
-                $ResponseContent.result.data | Select-Object -ExpandProperty 'url' | select -First 1 | % {
+                $ResponseContent.result.data | Select-Object -ExpandProperty 'url' | Select-Object -First 1 | ForEach-Object {
                     Write-Verbose ('Downloading image to {0}' -f $OutFile)
-                    Microsoft.PowerShell.Utility\Invoke-WebRequest `
-                        -Uri $_ `
-                        -Method Get `
-                        -OutFile $OutFile `
-                        -UseBasicParsing
+                    $params = @{
+                        Uri             = $_
+                        Method          = 'Get'
+                        OutFile         = $OutFile
+                        UseBasicParsing = $true
+                    }
+                    Invoke-WebRequest @param

```

Comment:
To avoid unintended code invocations, it is preferable to call the cmdlet with the module name explicitly.

```
Microsoft.PowerShell.Utility\Invoke-WebRequest @param
```